.gitignore

```diff
@@ -1,3 +1,5 @@
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
+build
```

Comment:
This should be `docs/build` and `docs/site` can be removed (since it's not being built with `mkdocs` any more).